stdlib/multiprocessing/reduction.pyi

```diff
@@ -27,9 +27,11 @@ def dump(obj, file, protocol: Any | None = ...) -> None: ...
 
 if sys.platform == "win32":
     if sys.version_info >= (3, 8):
-        def duplicate(handle, target_process: Any | None = ..., inheritable: bool = ..., *, source_process: Any | None = ...): ...
+        def duplicate(
+            handle: int, target_process: Any | None = ..., inheritable: bool = ..., *, source_process: Any | None = ...
+        ) -> int: ...
     else:
-        def duplicate(handle, target_process: Any | None = ..., inheritable: bool = ...): ...
+        def duplicate(handle: int, target_process: Any | None = ..., inheritable: bool = ...) -> int: ...
```

Comment:
```suggestion
        def duplicate(handle: int, target_process: Incomplete | None = ..., inheritable: bool = ...) -> int: ...
```

Reply:
This would instead make more sense to be: `int | None`

* `duplicate` is defined here: https://github.com/python/cpython/blob/main/Lib/multiprocessing/reduction.py#L71
* `target_process` is the third argument to `_winapi.DuplicateHandle` here: https://github.com/python/cpython/blob/main/Lib/multiprocessing/reduction.py#L79
* `_winapi.DuplicateHandle` is here, and only accepts `ints` https://github.com/python/typeshed/blob/master/stdlib/_winapi.pyi#L165

Reply:
Makes sense.